deployment/install-worker.sh

```diff
@@ -1,40 +1,69 @@
 #!/bin/bash
-set +e
 
+##############################################################################
+# Description:
+# This is a helper script to install the worker binaries on your Apache Spark cluster
+#
+# Usage:
+# ./install-script.sh <cloud-provider> <path-to-worker> <local-worker-installation-path>
+#
+# Sample usage:
+# ./install-script.sh 
+#   azure 
+#   abfs://<blobcontainer>@<gen2storageaccount>.dfs.core.windows.net/<path>/Microsoft.Spark.Worker.netcoreapp2.1.linux-x64-0.1.0.tar.gz 
```

Comment:
I am wondering if we should move to `wget` and let the user to point to our release on GitHub (verified on HDInsight). We don't have to do this in this PR, but we may want to track this if we want to simplify the experience.
cc: @suhsteve

Reply:
Added the capability.